presto-sqlserver/src/main/java/io/prestosql/plugin/sqlserver/SqlServerClient.java

```diff
@@ -121,7 +120,7 @@ protected void copyTableSchema(Connection connection, String catalogName, String
                         columnMapping.getType(),
                         columnMapping.getReadFunction(),
                         columnMapping.getWriteFunction(),
-                        DISABLE_UNSUPPORTED_PUSHDOWN));
+                        SIMPLIFY_UNSUPPORTED_PUSHDOWN));
```

Comment:
does`testLargeIn` serve as coverage for this?

Reply:
yes